### PR TITLE
Fix elb healthcheck when protocol is not http/https

### PR DESCRIPTION
**Pull request by kelibezhani**
ELB failed to create when protocol is TCP because it is not expecting a PathToPing but just TCP: port pair.

### CODE REVIEW

**Comment by pchand-amplify** [comment]
Has this been tested? unit tests?

File: disco_aws_automation/disco_elb.py [hunk 1]
```diff
@@ -56,9 +56,12 @@ def get_cname(self, hostclass, domain_name):
     def _setup_health_check(self, elb_name, health_check_url, instance_protocol, instance_port):
         if not health_check_url:
             logging.warning("No health check url configured for ELB %s", elb_name)
-            return
+            if instance_protocol in ('http', 'https'):
```
Comment (aoskotsky-amplify) [inline disco_aws_automation/disco_elb.py]: the protocol is upper case in our config files

File: disco_aws_automation/disco_elb.py [hunk 2]
```diff
@@ -56,9 +56,12 @@ def get_cname(self, hostclass, domain_name):
     def _setup_health_check(self, elb_name, health_check_url, instance_protocol, instance_port):
         if not health_check_url:
             logging.warning("No health check url configured for ELB %s", elb_name)
-            return
+            if instance_protocol in ('HTTP', 'HTTPS'):
```
Comment (jconstance-amplify) [inline disco_aws_automation/disco_elb.py]: Should probably make `instance_protocol` uppercase with `.upper()`, just to be sure.

**Comment by jconstance-amplify** [comment]
It should be possible to add a unit test for this, I would think. Also you will need to bump the version number again.

**Comment by jconstance-amplify** [comment]
LGTM. @pchand-amplify or @aoskotsky-amplify ?

**Comment by aoskotsky-amplify** [comment]
👍

**Pull request merged**